ns3_adapter/include/ns3_client.h

```diff
@@ -38,31 +40,21 @@ class NS3Client
     NS3Client();
     ~NS3Client();
 
-    /**
-    * @brief connects and send handshake message to MOSAIC CARMA Ambassador 
-    * @param message udp message
-    * @param remote_address IPv4 address of OBU
-    * @param remote_port of client service
-    * @param local_port of client service
-    * @return true on sucessful connect, false otherwise
-    */
-    bool registermsg(const std::shared_ptr<std::vector<uint8_t>>&message, 
-                const std::string &remote_address, unsigned short remote_port, 
-                unsigned short local_port);
-
     /**
     * @brief Connects the driver to the OBU at the provided IPv4 address and Port
     * @param address IPv4 address of OBU
-    * @param port of client service
+    * @param remote_broadcasting_port of server's v2x service
+    * @param remote_registration_port of server's registration service
+    * @param local_v2x_port of client v2x service
+    * @param local_time_port of client time service
     * @param ec error code set during connect
     * @return true on sucessful connect, false otherwise
     */
-    bool connect(const std::string &remote_address,unsigned short remote_port,
-                        unsigned short local_port,boost::system::error_code &ec);
+    bool connect_registration_and_broadcasting(const std::string &remote_address,unsigned short remote_broadcasting_port,unsigned short remote_registration_port,
+                        unsigned short local_v2x_port, unsigned short local_time_port, boost::system::error_code &ec);
 
 
-    bool connect(const std::string &remote_address, unsigned short remote_port,
-                 unsigned short local_port);
+    bool connect_registration(const std::string &remote_address, unsigned short remote_port);
```

Comment:
Is this method ever called. The one instance where it is called seems to be commented out or am I missing something.

Reply:
Yes correct. I forgot to delete. Please see the previous comment for the reasoning or let me know if there is more questions.